dandiapi/api/models/upload.py

```diff
@@ -50,18 +51,14 @@ class Meta:
     multipart_upload_id = models.CharField(max_length=128, unique=True, db_index=True)
     size = models.PositiveBigIntegerField()
 
-    @classmethod
-    def object_key(cls, upload_id):
-        upload_id = str(upload_id)
-        return (
-            f'{settings.DANDI_DANDISETS_BUCKET_PREFIX}'
-            f'blobs/{upload_id[0:3]}/{upload_id[3:6]}/{upload_id}'
-        )
+    @abstractclassmethod
+    def object_key(cls, upload_id, **kwargs):  # noqa: N805
+        ...
```

Comment:
We use `pass` elsewhere consistently in the codebase
```suggestion
        pass
```

Reply:
All the docs on the `@abstractmethod` page use `...` so I thought it was some magically syntax lol, I didn't know it was synonymous with `pass`.